rclcpp_examples/CMakeLists.txt

```diff
@@ -101,23 +101,36 @@ if(AMENT_ENABLE_TESTING)
     find_package("${middleware_impl}" REQUIRED)
   endforeach()
 
-  foreach(example_executable list_parameters_async list_parameters parameter_events_async parameter_events set_and_get_parameters_async set_and_get_parameters)
+  # Add each test case.  Multi-executable tests can be specified in
+  # semicolon-separated strings, like  exe1;exe2.
+  foreach(example_executable list_parameters_async list_parameters parameter_events_async parameter_events set_and_get_parameters_async set_and_get_parameters "talker;listener" "add_two_ints_client;add_two_ints_server" "add_two_ints_client_async;add_two_ints_server")
```

Comment:
This line seems to become excessively long. Can you create a list variable for this before and place every entry on a separate line to make it more readable?